wagtail/admin/templates/wagtailadmin/workflows/edit_task.html

```diff
@@ -51,22 +51,28 @@
                                 <em data-w-progress-target="label">{% trans 'Save' %}</em>
                             </button>
                         {% endfragment %}
-                        {% dropdown_button button=button toggle_icon="arrow-up" %}
-                            {% if can_enable %}
-                                <button
-                                    class="button"
-                                    data-controller="w-action"
-                                    data-action="w-action#post"
-                                    data-w-action-continue-value="true"
-                                    data-w-action-url-value='{{ view.get_enable_url }}'
-                                    type="submit"
-                                >
-                                    {{ view.enable_item_label }}
-                                </button>
-                            {% elif can_disable %}
-                                <a href="{{ view.get_delete_url }}" class="button">{{ view.delete_item_label }}</a>
-                            {% endif %}
-                        {% enddropdown_button %}
+                        {% if can_enable or can_disable %}
+                            {% dropdown_button button=button toggle_icon="arrow-up" %}
+                                {% if can_enable %}
+                                    <button
+                                        class="button"
+                                        data-controller="w-action"
+                                        data-action="w-action#post"
+                                        data-w-action-continue-value="true"
+                                        data-w-action-url-value='{{ view.get_enable_url }}'
+                                        type="submit"
+                                    >
+                                        {{ view.enable_item_label }}
+                                    </button>
+                                {% elif can_disable %}
+                                    <a href="{{ view.get_delete_url }}" class="button">{{ view.delete_item_label }}</a>
+                                {% endif %}
+                            {% enddropdown_button %}
+                        {% else %}
+                            <div class="w-grid">
```

Comment:
In other places we're adding a combination of `w-flex` and `w-flex-grow`. I assume this is to ensure the button fills the whole width of the container? I wonder if this is something that could/should be handled in the CSS instead, so we don't have to add the utility classes in different places (with different approaches) which might result in unwanted behaviour if we miss it.

Reply:
Yes, I was keen to further refactor this so we’d have a reusable "footer actions" component, but essentially ran out of time. Will switch to `w-flex-grow` and `w-flex` for now.